src/register/register-gnome/gnucash-header.c

```diff
@@ -606,6 +606,9 @@ gnc_header_init (GncHeader *header)
     header->width = 400;
     header->style = NULL;
 
+    // Set the style context for this widget so it can be easily manipulated with css
+    gtk_style_context_add_class (gtk_widget_get_style_context (GTK_WIDGET(header)), "GncRegisterHeader");
+
```

Comment:
Ideally we should be using
`gtk_widget_class_set_css_name` rather than `gtk_style_context_add_class`. Unfortunately this is only available as of gtk 3.20, so that's of little use now. It looks like the css code was in heavy flux in the range of gtk versions we currently support.
I'm not sure what is the best alternative. `gtk_style_context_add_class` or `gtk_widget_set_name`. For now I don't think this needs changing, but it's something to keep in mind for the future.